Project1/Stats.h

```diff
@@ -23,6 +26,7 @@ class Stats : Component<Unit> {
 	AttributeList& getAttribs();
 	AttributeList& getGrowths();
 	WeaponList& getWeaponProficiencies();
+	MobilityType& getMobilityType();
```

Comment:
Should return set of MobilityType objects.

Reply:
Ok, I'm currently working on doing this, but how should the movement formulas work? For each space, would I Iterate over the types of movement, see what I have that can enter, then use the one that costs the least? Iterating through an enum is awk.